C/plugins/common/omf.cpp

```diff
@@ -1427,12 +1426,16 @@ const std::string OMF::createTypeData(const Reading& reading)
 		tData.append("\": {\"type\": \"string\"},");
 	}
 
-	if (m_PIServerEndpoint.compare("c") == 0)
+	// Connector relay / ODS / EDS
+	if (m_PIServerEndpoint.compare(END_POINT_CR) == 0 ||
```

Comment:
If you use an enum then you can avoid the if else clauses and have a simple switch statement.

Reply:
done, thanks.